e2e/cypress/tests/beforeLoginTests.spec.ts

```diff
@@ -0,0 +1,19 @@
+//test ability to browse/download codebases
+describe("Browse Codebases", () => {
+  it("should browse the model library and be able to enter a search query", () => {
+    cy.visit("/"); // visit homepage
+    cy.visit("/codebases");
+    assert(cy.get("h1").contains("Computational Model Library"));
```

Comment:
should we also make assertions on an expected model to find in the model library?